roofit/roofitcore/CMakeLists.txt

```diff
@@ -230,6 +230,7 @@ ROOT_STANDARD_LIBRARY_PACKAGE(RooFitCore
     RooNaNPacker.h
     RooBinSamplingPdf.h
     RooBinWidthFunction.h
+    RooFit/Owner.h
```

Comment:
Is this a conscious decision to put this in `RooFit`? You're adding one more directory that has to be managed.

Reply:
Yes, that was a deliberate choice. We have too many headers now with terribly generic names like [Floats.h](https://github.com/root-project/root/blob/master/roofit/roofitcore/inc/Floats.h), [UniqueId.h](https://github.com/root-project/root/blob/master/roofit/roofitcore/inc/UniqueId.h), and not possible Owner.h.

Many distributors install the ROOT headers right into `/usr/include`, so it's bad to have these header file names I'd say. Fortunately it's not too late to do something about it, so I wanted to establish a convention to avoid name collisions:
* Installed RooFit headers must start with `Roo*` or must be located in a subdirectory starting with `Roo*` (e.g. RooFit or RooStats). 
* Similarly, if the class name doesn't start with Roo, it has to go in a `Roo*` namespace (usually `RooFit`)
* Free functions always need to go in this namespace
* For implmentation details that we can't avoid installing, we can use a `Roo*::Detail` namespace like we have with `ROOT::Detail` (same with `Experimental`)
I'd still have time to move Floats.h and and UniqueId.h into `RooFit`.

What are your thoughts on this?